pkg/doc/verifiable/common.go

```diff
@@ -85,6 +85,13 @@ func (r *DIDKeyResolver) resolvePublicKey(issuerDID, keyID string) (interface{},
 		}
 	}
 
+	// if key not found in PublicKey try to find it in authentication
```

Comment:
I'm a bit confused, how did you decide priority ordering?

Reply:
priority not important here because we search by key id

Reply:
I think we will need to revisit.